backend/utils/log_events.py

```diff
@@ -38,13 +44,98 @@ def disconnect(sid: str) -> None:
     logger.info(f"[{os.getpid()}] Client with SID:{sid} disconnected")
 
 
-def _handle_pubsub_messages(message: Any) -> None:
-    channel = message["channel"].decode("utf-8")
-    data = message["data"]
+def _get_validated_log_data(json_data: Any) -> Optional[LogDataDTO]:
+    """Validate log data to persist history
+    Args:
+        json_data (Any): Log data in JSON format
+    """
+    print(f"Received log data: {json_data}")
```

Comment:
@muhammad-ali-e change this to a debug log

Reply:
UPDATED